frontend/src/components/header/header.tsx

```diff
@@ -47,6 +48,16 @@ export const Header = (): React.ReactElement => {
 
   const { data: user } = useGetUserQuery();
 
+  useEffect(() => {
+    if (location.pathname === AppRoute.SEARCH) {
+      setValue(0);
+    } else if (location.pathname === AppRoute.ABOUT_US) {
+      setValue(1);
+    } else {
+      setValue(-1);
+    }
+  });
```

Comment:
This should be in the `switch` statement as we have comparisons of the static values. What does the numeric value even means? 

To display that smth active we need to store the active route string